heapster.go

```diff
@@ -35,8 +35,7 @@ import (
 )
 
 var (
-	argPollDuration    = flag.Duration("poll_duration", 10*time.Second, "The frequency at which heapster will poll for stats")
-	argStatsResolution = flag.Duration("stats_resolution", 5*time.Second, "The resolution at which heapster will retain stats. Acceptable values are in the range [1 second, 'poll_duration')")
+	argStatsResolution = flag.Duration("stats_resolution", 5*time.Second, "The resolution at which heapster will retain stats.")
```

Comment:
Without resolution, we will end up handling metrics of 1 second granularity. Is this something we want? It will lead to additional memory usage?